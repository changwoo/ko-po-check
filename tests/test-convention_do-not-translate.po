# -*-text-*-
# convention/do-not-translate
#
msgid ""
msgstr ""
"Project-Id-Version: test do-not-translate\n"
"POT-Creation-Date: 2018-04-06 16:09 +0900\n"
"PO-Revision-Date: 2018-04-06 16:09 +0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: GNOME Korea <gnome-kr@googlegroups.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;"

#. TRANSLATORS: Do NOT translate or transliterate this text!
#. This is an icon file name
#: data/blahblah.desktop.in.in:9
msgid "blah"
msgstr "블라"

#. TRANSLATORS: Do NOT translate or transliterate this text!
#. This is an icon file name
#: data/blahblah.desktop.in.in:9
msgid "blahblah"
msgstr "blahblah"
