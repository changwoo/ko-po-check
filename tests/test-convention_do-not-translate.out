-:20: 번역하지 말라는 표시가 되어 있는 원문을 번역했습니다
